### ISSUE TITLE
GroupedList - child component cannot be focused with keyboard navigation

### ISSUE THREAD

**Issue by akvangari** [microsoft/fluentui#20614]
### Environment Information

- **Package version(s)**: 8.11.0
- **Browser and OS versions**: macOS Big Sur 11.6, Chrome Version 95.0.4638.69

### Describe the issue:

When using the `GroupedList1 component, the children don't go into focus when pressing the Tab key on the keyboard. In turn, this is blocking us due to a11y requirements failing. 

### Please provide a reproduction of the bug in a codepen:
https://codesandbox.io/s/adoring-goldwasser-5ct2n

#### Actual behavior:
After the first child is focused, pressing tab skips over the remaining children. 

#### Expected behavior:
Pressing the tab key should cycle through all the children.

**Comment by gouttierre** [comment]
@akvangari  - Thanks for filing this issue with us.  To set expectations, the developers will review this issue once capacity allows. I do see tabbing focuses on the first child, but pressing tab again skips over the remaining children, unless the users uses the arrow down key.
![image](https://user-images.githubusercontent.com/78806055/148227606-b446e1f5-b8f4-432c-8d54-1eb9981330d3.png)

@spmonahan - Would you be able to confirm if this is a regression, or if this behavior is an issue?

**Comment by spmonahan** [comment]
Based on [ARIA authoring practices for keyboard interaction](https://www.w3.org/TR/wai-aria-practices-1.1/#keyboard-interaction-23) I believe this is a bug.

**Comment by cbhumireddy441** [comment]
@gouttierre @spmonahan, can you please suggest is there any chance to know when this issue will be fixed? We have a similar use case, where a group of controls including text boxes (which expects input from user). Tab is not focusing controls on Grouped List.
